src/document.jl

```diff
@@ -1,9 +1,9 @@
 
 #### Document type
 
-immutable _XMLDocStruct  # use the same layout as C
+struct _XMLDocStruct  # use the same layout as C
```

Comment:
You'll need to set the REQUIRE file to `julia 0.6`

Reply:
You should also update the Travis CI and AppVeyor file

Reply:
All that should be done now.